kanidmd/daemon/Cargo.toml

```diff
@@ -18,9 +18,11 @@ name = "kanidmd"
 path = "src/main.rs"
 
 [dependencies]
-kanidm.workspace = true
+# Remove these!
```

Comment:
orly? like now, or later?

Reply:
It's in progress!

Reply:
Didn't know if it was a leftover or not 😄